MyWeatherApp/Models/ForecastViewModel.swift

```diff
@@ -54,17 +60,29 @@ final class ForecastViewModel: ObservableObject {
         }
     }
 
-    func loadingTask() {
-        if appData.city.isEmpty {
-            defaultCityName = "Москва"
-        } else {
-            defaultCityName = appData.city
-        }
-        fetchWeather(city: defaultCityName)
-    }
-
     func searchTask() {
         fetchWeather(city: query)
         query = ""
     }
+
+    init(){
+        locationManager.$status.sink { [self] status in
+            if status == nil {
+                return
+            } else {
```

Comment:
Используем стиль fast return: он позволяет сократить вложенность кода, чем повышает удобство чтения

```suggestion
            if status == nil {
                return
            }
            
            switch status {
            ...
            }
```